### ISSUE TITLE
Research: OpenSearch and Apache Lucene

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#3]
## User story
1. As a PO
2. I need skilled developers that know the tools they have to work with
3. So that productive work on the project can start as soon as possible

## Acceptance criteria
* Get familiar with [OpenSearch](https://opensearch.org/docs/latest/)
* Get familiar with [Apache Lucene](https://lucene.apache.org/)
* Don't forget the links and slides that Mike provided us with

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
